### ISSUE TITLE
TR-4774 Address Travis-CI changes causing django-elastic-migrations test failures

### ISSUE THREAD

**Issue by codekiln** [HBS-HBX/django-elastic-migrations#79]
At some point between [#30 multiprocessing tests](https://github.com/HBS-HBX/django-elastic-migrations/pull/63) the next merged PR, [Upgrading to Django 2.2](https://github.com/HBS-HBX/django-elastic-migrations/pull/68), Travis-CI must have updated its default images, causing tests to fail:

```
 * Starting Elasticsearch Server
warning: Falling back to java on path. This behavior is deprecated. Specify JAVA_HOME
/usr/share/elasticsearch/bin/elasticsearch-env: line 71: /etc/default/elasticsearch: Permission denied
   ...fail!
The command "sudo service elasticsearch start" failed and exited with 1 during .
```

 See also [this comment](https://github.com/HBS-HBX/django-elastic-migrations/pull/75#pullrequestreview-400120665).

It's probably [as simple as updating these lines](https://github.com/HBS-HBX/django-elastic-migrations/blob/c672138f90df395a77282343d0b5b0665c09739f/.travis.yml#L41-L45).

**Comment by kofic** [comment]
Nice catch @codekiln

**Comment by esmail** [comment]
Closed by #80 ?
